django_extensions/management/commands/runserver_plus.py

```diff
@@ -222,6 +222,13 @@ def postmortem(request, exc_type, exc_value, tb):
                 print("Exception occured: %s, %s" % (exc_type, exc_value), file=sys.stderr)
                 p.post_mortem(tb)
 
+        # Disable threading by default if debugger options are used.
+        if options["threaded"] is None:
+            if pdb_option or ipdb_option or pm:
+                options["threaded"] = False
```

Comment:
Thought about having a `print` there, but it should also get displayed once at most, which is not trivial it seems.

> print('Disabling threading with debugger, use --threaded to force.')

Reply:
I like the idea of having the `print` there !

We should use:
```python
if self.show_startup_messages:
    print('Disabling threading with debugger, use --threaded to force.')
```

Reply:
I've tried that, but it did not work out like I've expected, but could retry / check it again.

Reply:
> I've tried that, but it did not work out like I've expected, but could retry / check it again.

What was the issue with the `print` statement ?

Reply:
I think it was printed always (on reload) anyway - but might have been related to using this in another place / or it needs to be moved for this to work.

Reply:
Would be nice if the output was there... (Hopefully it's not too difficult to add)
In any case it would be good if we update the `runserver_plus` docs as well and fix the small style issue below.

Let's get this merged :-)